app/src/main/java/org/sopt/kream/presentation/ui/main/home/recommend/RecommendCircleMenuAdapter.kt

```diff
@@ -0,0 +1,29 @@
+package org.sopt.kream.presentation.ui.main.home.recommend
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.kream.databinding.ItemRecommendCircleMenuBinding
+import org.sopt.kream.presentation.ui.type.RecommendCircleMenuType
+import kotlin.enums.EnumEntries
+
+class RecommendCircleMenuAdapter(private val list: EnumEntries<RecommendCircleMenuType>) : RecyclerView.Adapter<RecommendCircleMenuViewHolder>() {
+    private val item = list
+
+    override fun onCreateViewHolder(
+        parent: ViewGroup,
+        viewType: Int,
+    ): RecommendCircleMenuViewHolder {
+        val binding = ItemRecommendCircleMenuBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return RecommendCircleMenuViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(
+        holder: RecommendCircleMenuViewHolder,
+        position: Int,
+    ) {
+        holder.onBind(item.get(position).menu)
```

Comment:
```suggestion
        holder.onBind(item[position].menu)
```
그리고 이거 이미지 디쟌 측에서 추가해주시면 menu만 넘기는 게 아니라 그냥 아이템 객체 자체를 넘겨야 할 것 같숩니다

Reply:
넵~~!!